ocs_ci/ocs/pgsql.py

```diff
@@ -207,6 +208,25 @@ def get_pgbench_status(self, pgbench_pod_name):
             'terminated'
         ]['reason']
 
+    def get_postgresql_status(self, status=constants.STATUS_RUNNING):
+        """
+        Get all postgresql pods on cluster
+        Check that all postgresql pods in desired state
+
+        Args:
+           status (str): Desired state of postgresql pods
+
+        Returns:
+            str: state of postgresql pod
+
+        """
+        postgresql_pods = self.get_postgresql_pods()
+
+        for pod in postgresql_pods:
+            wait_for_resource_state(
+                resource=pod, state=status, timeout=300
```

Comment:
if something goes wrong, this means that potentially the test will wait for 300 seconds multiplied by the number of pods. So if we have 10 pods which are all stuck in Pending or Terminating, we will wait for 3000 seconds